sensing/src/main/java/com/google/android/sensing/model/UploadRequest.kt

```diff
@@ -25,13 +26,14 @@ data class UploadRequest(
   val resourceInfoId: String,
   val zipFile: String,
   val fileSize: Long,
-  var fileOffset: Long,
+  var fileOffset: Long = 0,
```

Comment:
Can you move fileOffset below (your call where it fits among other variables) so that all mandatory fields are on top and default value fields are at bottom ?